GWFish/detectors.yaml

```diff
@@ -40,6 +42,7 @@ CE1:
             spacing:          geometric 
             df:               1./4.    
             npoints:          5000 
+            arm_length:       40000
```

Comment:
I am not sure  about the arm length. If I remember correctly, CE1 is the less sensitive concept, and if I am not mistaken, it was meant to represent a 20km detector. I could be wrong though! Please check to make sure.